frontend/src/components/ChangePassword.js

```diff
@@ -0,0 +1,271 @@
+import React, { useContext, useState } from "react";
+import config from "../config.json";
+import qs from "qs";
+import {
+  Button,
+  Column,
+  Form,
+  FormLabel,
+  Grid,
+  Heading,
+  ListItem,
+  Loading,
+  PasswordInput,
+  Section,
+  Stack,
+  TextInput,
+  UnorderedList,
+} from "@carbon/react";
+import { FormattedMessage, injectIntl, useIntl } from "react-intl";
+import { Formik } from "formik";
+import * as Yup from "yup";
+import { AlertDialog, NotificationKinds } from "./common/CustomNotification";
+import { NotificationContext } from "./layout/Layout";
+
+function ChangePassword() {
+  const intl = useIntl();
+  const { notificationVisible, addNotification, setNotificationVisible } =
+    useContext(NotificationContext);
+  const [submitting, setSubmitting] = useState(false);
+
+  const changePassword = (values) => {
+    setSubmitting(true);
+    // Call the API to change the password show success or error notification on success redirect to the login page
+    fetch(config.serverBaseUrl + "/ChangePasswordLogin", {
```

Comment:
Thanks @rahulkundu1209 .
You need to re-write the [ChangePassowrd Controller ](https://github.com/I-TECH-UW/OpenELIS-Global-2/blob/develop/src/main/java/org/openelisglobal/login/controller/ChangePasswordLoginController.java) into a REST-full controller . not that the naming paater shoulbe be `/rest/ChangePasswordLogin`

Reply:
Hello @rahulkundu1209 , i have tested this and the Old controller works well. It can handle the hson form